tests/python/common/data/test_data.py

```diff
@@ -395,6 +395,25 @@ def test_zinc():
         assert not label.shape
 
 
+@unittest.skipIf(
```

Comment:
Please verify the test can pass locally with dependency and move it to tests/integration/test_data.py, we will handle the running in regression framework later in the future.

Reply:
should I remain the test function in [tests/python/common/data/test_data.py](https://github.com/dmlc/dgl/pull/6337/files/be5666930404c139a1a243786530218e8c115b7a#diff-a9dac15a841615410378677e848df85edbffb5da85dbaeb8c7e23672a21eb656), or just move it to tests/integration/test_data.py

Reply:
just move, if you put it here, it will fail CI since we didn't setup the dependency.
By moving it to tests/integration/test_data.py, in the future we will setup the dependency in integrating test env instead of CI env.

Reply:
fix it, please help me to review it